### PR TITLE
my bad

### PR DESCRIPTION
**Pull request by Slyrith**
my dumbass forgot to update the index while having the format wrong, i need to drink water fr

### CODE REVIEW

**Review by Neodevils — APPROVED** [review]
LGTM.

File: src/index.ts
Comment (Neodevils) [inline src/index.ts:1]: Exporting as `messageId.js` is fine.

File: src/Functions/Message/messageId.js
Comment (Neodevils) [inline src/Functions/Message/messageId.js:1]: Well, the file has to end with `.ts` extension.

**Review by Neodevils — CHANGES_REQUESTED** [review]
Renaming file extension would be better for handling.

File: src/Functions/Message/messageId.js
Comment (Slyrith) [reply src/Functions/Message/messageId.js:1]: yep, check commit already fixed it

Comment (Neodevils) [reply src/Functions/Message/messageId.js:1]: What? It is still messageId.js as file name. 😂

**Comment by Neodevils** [comment]
IMG_0900

File: src/Functions/Message/messageId.js
Comment (Slyrith) [reply src/Functions/Message/messageId.js:1]: im trippin

**Comment by Slyrith** [comment]
im trippin 🥴

**Pull request merged**